examples/render-helm-chart-local/.expected/exec.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-kpt fn eval --image-pull-policy ifNotPresent --image gcr.io/kpt-fn/inflate-helm-chart:unstable \
+kpt fn eval --image-pull-policy ifNotPresent --image gcr.io/kpt-fn/render-helm-chart:unstable \
```

Comment:
> --image-pull-policy ifNotPresent

Can you please ensure all `render-helm-chart` examples are using `--image-pull-policy never`

Reply:
sorry about that, checked them all and updated